variants/STM32F4xx/F405RGT_F415RGT/PeripheralPins_MICROMOD_F405.c

```diff
@@ -0,0 +1,349 @@
+/*
```

Comment:
Seems based on older generated file.
I would recommend to use the default PeripheralPins.c as base for this file.
This would prevent issue if some change done since the base file you used.